### PR TITLE
[Adjustment][Vince] dropdown navbar on hover

### PR DESCRIPTION
**Pull request by VinceRaAle**
https://www.loom.com/share/1eaf23f251894d0ba3f71eb8162d6dc1

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -18,18 +18,32 @@ function Header() {
...
+  function handleHoverEnter() {
+    setIsAboutOpen(true);
+  }
+
+  function handleHoverLeave() {
+    if (!isInside) {
+      setIsAboutOpen(false);
+    }
+  }
+
```
Comment (ezerssss) [inline components/Header.tsx]: pwede na ni tangalon kay ang useEffect na ang ga manage sa hovering man guro

File: components/Header.tsx [hunk 2]
```diff
@@ -81,7 +95,13 @@ function Header() {
         </Link>
         <div className="hidden md:flex gap-10 items-center">
           <div className="relative" ref={ref}>
-            <button onClick={handleAboutClick}>About Us</button>
+            <button
+              onClick={handleAboutClick}
+              onMouseEnter={handleHoverEnter}
+              onMouseLeave={handleHoverLeave}
```
Comment (ezerssss) [inline components/Header.tsx]: matangal nani duha

File: hooks/useMousePosHook.tsx [hunk 3]
```diff
@@ -1,27 +1,43 @@
 import { useEffect, useState } from "react";
 
 function useMousePosHook(ref: React.RefObject<any>) {
-  const [isInside, setIsInside] = useState(false);
+  const [isHoverInside, setIsHoverInside] = useState(false);
 
-  function handleClickOutside(event: MouseEvent) {
+  function handleHover(event: MouseEvent) {
```
Comment (ezerssss) [inline hooks/useMousePosHook.tsx:25]: ayaw to i delete ang `handleClickOutside` nga function, i add lang ni ang `handleHover`, so basically naa tay handleClickOutside AND handleHover

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: hooks/useMousePosHook.tsx (hunk 3 above)
Comment (VinceRaAle) [reply hooks/useMousePosHook.tsx:25]: resolved

File: components/Header.tsx (hunk 2 above)
Comment (VinceRaAle) [reply components/Header.tsx]: resolved

File: components/Header.tsx (hunk 1 above)
Comment (VinceRaAle) [reply components/Header.tsx]: resolved

**Review by ezerssss — APPROVED** [review]
lgtm

**Pull request merged**